lalrpop/src/api/mod.rs

```diff
@@ -208,21 +212,24 @@ impl Configuration {
     /// Process all files in the current directory, which -- unless you
     /// have changed it -- is typically the root of the crate being compiled.
     pub fn process_current_dir(&self) -> Result<(), Box<dyn Error>> {
+        // If we can get a current dir, check to make sure session.in_dir either *wasn't* set, or
+        // wasn't set to that dir.  If we can't get a current dir, we'll error out in a moment
+        // anyways, and that's the bigger problem.
+        if let Ok(current_dir) = current_dir() {
+            if self.session.in_dir.is_some() && self.session.in_dir != Some(current_dir) {
+                eprintln!("Error: \"process_current_dir()\" contradicts previously set in_dir");
```

Comment:
Is there a typical fix that we expect users to make? Like drop `set_in_dir(P)` and maybe update `set_out_dir`? I would like to give a more actionable error message if possible

Reply:
Updated test.  Will push in a moment.

Reply:
Thanks!